src/ForumReader.ts

```diff
@@ -58,15 +58,16 @@ export default class ForumReader {
         bot.on("ready", () => {
             const guild = bot.guilds.get(this.sharedSettings.server);
             if (!guild) {
-                console.error(`ForumReader: Incorrect settings for guild ID ${this.sharedSettings.server}`);
+                console.error(`ForumReader: Unable to find server: ${this.sharedSettings.server}`);
```

Comment:
Would still point out that they're IDs, this could be interpreted as server names